test/jdk/javax/accessibility/JSlider/AccessibleAction/JSliderAccessibleAction.java

```diff
@@ -0,0 +1,286 @@
+                            + jSliderInitialValue + "  currentJSliderValue = " + currentJSliderValue);
+                }
+
+                mouseAction(robot, invalidIncrementBtn);
+                if (!invalidIncrementCountDownLatch.await(30,
+                        TimeUnit.SECONDS)) {
+                    throw new RuntimeException("Failed to perform action on Invalid " +
+                            "Increment button");
+                }
+                if (jSliderInitialValue.get() != currentJSliderValue.get()) {
+                    throw new RuntimeException("Expected that JSlider value is not " +
+                            "changed when invalid decrement value -1 is passed to " +
+                            "doAccessibleAction(-1)  jSliderInitialValue = "
+                            + jSliderInitialValue + "  currentJSliderValue = " + currentJSliderValue);
+                }
+
+                // JSlider value is decremented
+                mouseAction(robot, decrementBtn);
+                if (!validDecrementCountDownLatch.await(30, TimeUnit.SECONDS)) {
+                    throw new RuntimeException("Failed to perform action on valid " +
+                            "decrement button");
+                }
+                if (jSliderInitialValue.get() == currentJSliderValue.get()) {
+                    throw new RuntimeException("Expected that JSlider value is  " +
+                            "decremented when value 1 is passed to " +
+                            "doAccessibleAction(1)  jSliderInitialValue = "
+                            + jSliderInitialValue + "  currentJSliderValue = " + currentJSliderValue);
+                }
+
+                // JSlider value is incremented
+                mouseAction(robot, incrementBtn);
+                if (!validIncrementCountDownLatch.await(30, TimeUnit.SECONDS)) {
+                    throw new RuntimeException("Failed to perform action on valid " +
+                            "Increment button");
+                }
+                if (jSliderInitialValue.get() != currentJSliderValue.get()) {
+                    throw new RuntimeException("Expected that JSlider value is  " +
+                            "incremented when value 0 is passed to " +
+                            "doAccessibleAction(0)  jSliderInitialValue = "
+                            + jSliderInitialValue + "  currentJSliderValue = " + currentJSliderValue);
+                }
+            } finally {
+                SwingUtilities.invokeAndWait(() -> {
+                    if (jFrame != null) {
+                        jFrame.dispose();
+                    }
+                });
+            }
+        }
+    }
+
+    public static void mouseAction(Robot robot, JButton button) throws InterruptedException,
+            InvocationTargetException {
+        robot.waitForIdle();
+        final Point[] point = new Point[1];
+        final Rectangle[] rect = new Rectangle[1];
```

Comment:
finals here are redundant although do not hurt

Reply:
Fixed